src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private final int number;
+    private final String color;
+
+    public Ball(int number, String color) {
+        this.number = number;
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "№ " + number + " - " + color;
```

Comment:
you would suggest adding in the toString method additional contextual data about the kind of object you have.